document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -210,7 +210,10 @@ protected String parseNonCompositeFilter(Filter filter, Params.Builder paramsBui
     String fieldName = filter.getFieldName();
     String fullFieldName = prepareCast(getFieldPrefix(fieldName), value);
     StringBuilder filterString = new StringBuilder(fullFieldName);
+    Boolean isNotOperator = false;
     switch (op) {
+      case NEQ:
```

Comment:
I see that we also don't have support for this on the mongo side as well, we have open PR - https://github.com/hypertrace/document-store/pull/20. So, would suggest keeping this filter `Un-supported` for parity or incorporating it together.

Reply:
Sure, removed this for now